changelog/9396.bugfix.rst

```diff
@@ -0,0 +1 @@
+Ensure :attr:`pytest.Config.inifile` is available during the :func:`pytest_cmdline_main <_pytest.hookspec.pytest_cmdline_main>` hook.
```

Comment:
Maybe clarify this only fixes a regression in `7.0.0rc1`, i.e. not relevant for most users.

Reply:
Good idea, done!